### ISSUE TITLE
linktype="examples"のリンク修復

### ISSUE THREAD

**Issue by momdo** [waic/wcag20#175]
以下のようにリンクが切れているものがあるので、修復する。
http://waic.github.io/wcag20/Techniques/F91.html#F91-description
https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/F91#F91-examples

https://github.com/waic/wcag20/blob/3ea04843fd6977c9c523727adc0391f1e0d2555f/wcag20/sources/techniques/failures/F91.xml#L85-L87

**Comment by bakera** [comment]
とりあえずリンクは修正する方向で。
リンクテキストも和訳すべきですが、これについてはいったん保留し、ひとまずリンクがつながれば良いとします。

**Comment by momdo** [comment]
リンク自体は修復できているので、閉じます。（WCAG 2.1関連文書での扱いは別途考えることに。）
